neo/Network/RPC/ContractClient.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Neo.Network.P2P.Payloads;
+using Neo.Network.RPC.Models;
+using Neo.SmartContract;
+using Neo.SmartContract.Manifest;
+using Neo.VM;
+using Neo.Wallets;
+
+namespace Neo.Network.RPC
+{
+    /// <summary>
+    /// Contract related operations through RPC API
+    /// </summary>
+    public class ContractClient
+    {
+        protected readonly RpcClient rpcClient;
+
+        /// <summary>
+        /// ContractClient Constructor
+        /// </summary>
+        /// <param name="rpc">the RPC client to call NEO RPC methods</param>
+        public ContractClient(RpcClient rpc)
+        {
+            rpcClient = rpc;
+        }
+
+        /// <summary>
+        /// Generate scripts to call a specific method from a specific contract.
+        /// </summary>
+        /// <param name="scriptHash">contract script hash</param>
+        /// <param name="operation">contract operation</param>
+        /// <param name="args">operation arguments</param>
+        /// <returns></returns>
+        public static byte[] MakeScript(UInt160 scriptHash, string operation, params object[] args)
+        {
+            using (ScriptBuilder sb = new ScriptBuilder())
```

Comment:
This method `MakeScript` is not relative to RPC, i think that should be moved to `VM\Helper.cs`

Reply:
👌Moved.